mentors/delphi/n.json

```
@@ -1 +1,10 @@
-[]
+[
+  {
+    "github_username": "Neftali",
+    "name": "Germán Estévez -Neftalí-",
+    "link_text": Personal website about Delphi,
```

```suggestion
    "link_text": "Personal website about Delphi",
```
Same problem as before.  String values must be captured in `"`.